packages/aws-cdk/test/api/hotswap/appsync-mapping-templates-hotswap-deployments.test.ts

```diff
@@ -0,0 +1,308 @@
+import { AppSync } from 'aws-sdk';
+import * as setup from './hotswap-test-setup';
+
+let hotswapMockSdkProvider: setup.HotswapMockSdkProvider;
+let mockUpdateResolver: (params: AppSync.UpdateResolverRequest) => AppSync.UpdateResolverResponse;
+let mockUpdateFunction: (params: AppSync.UpdateFunctionRequest) => AppSync.UpdateFunctionResponse;
+let mockListFunctions: (params: AppSync.ListFunctionsRequest) => AppSync.ListFunctionsResponse;
+
+beforeEach(() => {
+  hotswapMockSdkProvider = setup.setupHotswapTests();
+  mockUpdateResolver = jest.fn();
+  mockUpdateFunction = jest.fn();
+  mockListFunctions = jest.fn().mockReturnValue({ functions: [{ name: 'my-function', functionId: 'functionId' }] });
```

Comment:
Hmm. Can we move this to the actual tests that require this? Because right now, we have assertions in the tests that verify we're calling the `updateFunction` API with the function ID `functionId`, but that value is not present anywhere in the test itself - which makes them a little hard to read.

If you move this stubbing to the test that actually uses it, it will make it very clear where does the value `functionId` come from.